core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/UpdateNodeCountValidator.java

```diff
@@ -170,6 +177,17 @@ public void validateScalabilityOfInstanceGroup(Stack stack, String instanceGroup
                 instanceGroup);
     }
 
+    public void validateInstanceGroupForStopStart(Stack stack, String instanceGroupName, int scalingAdjustment) {
+        Set<String> computeGroups = getComputeHostGroup(stack);
+        if (!computeGroups.contains(instanceGroupName)) {
+            if (upscaleEvent(scalingAdjustment)) {
+                throw new BadRequestException(format("Start instances operation is not allowed for %s host group.", instanceGroupName));
+            } else if (downScaleEvent(scalingAdjustment)) {
+                throw new BadRequestException(format("Stop instances operation is not allowed for %s host group.", instanceGroupName));
+            }
```

Comment:
Nit: else throw an exception if not an upscale or a downscale (in the scenario that this somehow ends up being called with a scalingAdjustment of 0).
The method would otherwise allow the operation, even though it would be harmless.